src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
Remember about correct indentation between parent and child elements. The div and its children should be indented with 2 spaces.